start-server.sh

```diff
@@ -22,14 +22,14 @@ nginx
 
 su ${USER}
 
-export OCT_DJANGO_PORT=4010
-export OCT_BASE_DIR="/plugin_data"
-export TRANSFORMERS_CACHE="/models/huggingface"
-export TRANSFORMERS_OFFLINE="0"
-export EASYOCR_MODULE_PATH="/models/easyocr"
-export TESSERACT_PREFIX="/models/tesseract"
-export TESSERACT_ALLOW_DOWNLOAD="true"
-export PADDLEOCR_PREFIX="/models/paddleocr"
+export OCT_DJANGO_PORT=${OCT_DJANGO_PORT:-4010}
+export OCT_BASE_DIR=${OCT_BASE_DIR:-"/plugin_data"}
```

Comment:
Also allowing the used to change this is not really useful as this is an internal variable of the container.
The external customization comes from what you bind mount to this location